src/utils/misc.ts

```diff
@@ -58,10 +58,7 @@ export const inIframe = () => {
 
 export const inLedgerIframe = () => {
   try {
-    return (
-      window.self !== window.top &&
-      (window?.location?.ancestorOrigins?.[0]?.includes('ledger') || window?.navigator?.userAgent.includes('Firefox'))
-    );
+    return window.self !== window.top && window?.location?.ancestorOrigins?.[0]?.includes('ledger');
```

Comment:
maybe we can try document.referrer for firefox?

Reply:
we need to check the string value on ledger and gnosis to see if we can work with it tho
https://developer.mozilla.org/en-US/docs/Web/API/Document/referrer

Reply:
will be empty if is not an iframe

Reply:
also maybe check window.location.origin

Reply:
Since ledger live is an installed app in the OS, and not really loaded through a browser, can we assume this condition is not needed?